packages/apollos-data-prayer/src/prayer-requests/data-source.js

```diff
@@ -155,53 +152,53 @@ export default class PrayerRequest extends RockApolloDataSource {
       .find(id)
       .get();
 
-  getFromIds = (ids) =>
-    this.request().filterOneOf(ids.map((id) => `Id eq ${id}`));
+  getFromIds = async (ids) => {
+    const idsFilter = ids.map((id) => `Id eq ${id}`);
+    return this.request()
+      .filterOneOf(idsFilter)
+      .get();
+  };
 
   getSavedPrayers = async () => {
     const {
       dataSources: { Followings },
     } = this.context;
 
-    try {
```

Comment:
Same thing about the `try ... catch`. Better error handling instead of removing it? Would it be important to know that there was an error getting data?

Reply:
same as above